scripts/publish.rs

```diff
@@ -0,0 +1,318 @@
+//! Helper script to publish the wasmtime and cranelift suites of crates
+//!
+//! Usage:
+//!
+//! * First, compile this script
+//! * Next, set cwd to the root of the wasmtime repository
+//! * Execute `./publish bump` to bump versions
+//! * Send a PR
+//! * Merge when green
+//! * Execute `./publish publish` to publish crates
```

Comment:
This usage comment is kind of a less-complete version of the docs in contributing-release-process.md. Would it make sense to just link to that document, and then make the comment here just briefly describe the CLI options?